app/models/measurement.rb

```diff
@@ -10,11 +9,8 @@ class Measurement < ActiveRecord::Base
 
   #input: measurement
   def bmi_calc
```

Comment:
I realize this was already here, but as I understand it you're calculating the BMI every time you want to show it for a particular measurement. Could you figure out a way to store the bmi in the database with the measurement and update it when/if needed?